src/ranged.cpp

```diff
@@ -1219,6 +1219,19 @@ void practice_archery_proficiency( player &p, const item &relevant )
     }
 }
 
+// Apply stamina cost to archery which decreases due to proficiency
+void mod_stamina_archery( player &p )
+{
+    // Set activity level if it's below that level
+    p.set_activity_level( ACTIVE_EXERCISE );
+
+    int stam_cost = 4;
+    stam_cost -= p.has_proficiency( proficiency_prof_bow_basic );
+    stam_cost -= p.has_proficiency( proficiency_prof_bow_expert );
+    stam_cost -= p.has_proficiency( proficiency_prof_bow_master );
```

Comment:
Could you add this as a proficiency bonus instead?
Though you might want to have a sanity check here about not giving the character stamina for drawing bows.

Reply:
This will no longer be relevant as per I-am-Erk's suggestion so I'm marking as resolved